liveviz/src/components/trackermap/ui/TrackerUI.js

```diff
@@ -71,15 +71,19 @@ class TrackerUI extends Component {
     this.state = {
       width: 0,
       height: 0,
+      arenaWidth: this.props.room.Data[0][0],
```

Comment:
The arena dimensions and sensor positions should not be on the state. They should be retrieved in the `render` function.